src/box/lua/upgrade.lua

```diff
@@ -1365,9 +1365,85 @@ local function upgrade_to_2_11_1()
     drop_schema_max_id()
 end
 
+--------------------------------------------------------------------------------
+-- Tarantool 2.11.4
+--------------------------------------------------------------------------------
+local function create_persistent_gc_state(issue_handler)
```

Comment:
1. If we acquired ddl on replace in `_gc_consumers`, user's transactions would be aborted for no apparent reason when the consumers are advanced (that's what "acquire ddl" actualy do). 
2. Since replace in `_gc_consumers` can be done from on_replace trigger of `_cluster`, limiting depth of its triggers would lead to error in such cases.

Added the reasons to commit message.